doc/howto/instances_console.md

```diff
@@ -0,0 +1,28 @@
+---
+discourse: 9223
+---
+
+(instances-console)=
+# How to access the console
+
+Use the `lxc console` command to attach to instance consoles.
+The console is available at boot time already, so you can use it to see boot messages and, if necessary, debug startup issues of a container or VM.
+
+To show log output, pass the `--show-log` flag:
+
+    lxc console <instance_name> --show-log
+
+## Log on to your virtual machine
```

Comment:
Can we call this just "Graphical console for virtual machines" or similar, as the "log on" bit isn't specific to graphical consoles.

Reply:
I think we should mention the need for installing the spice client for vga console.